<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComboDialog</class>
 <widget class="QDialog" name="ComboDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">#Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1" rowspan="2">
      <widget class="QPlainTextEdit" name="editSnippet">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="GroupComboBox" name="comboGroup">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonNewGroup">
         <property name="text">
          <string>&amp;New</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelKeyword">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#258fc0; vertical-align:super;&quot;&gt;● &lt;/span&gt;Keyword&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelGroup">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#258fc0; vertical-align:super;&quot;&gt;● &lt;/span&gt;Group&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSnippet">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#258fc0; vertical-align:super;&quot;&gt;● &lt;/span&gt;Snippet&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editKeyword"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="spacer_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#258fc0; vertical-align:super;&quot;&gt;● &lt;/span&gt;Required fields&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupComboBox</class>
   <extends>QComboBox</extends>
   <header>Group/GroupComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>comboGroup</tabstop>
  <tabstop>buttonNewGroup</tabstop>
  <tabstop>editKeyword</tabstop>
  <tabstop>editSnippet</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ComboDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editKeyword</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ComboDialog</receiver>
   <slot>updateGui()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editSnippet</sender>
   <signal>textChanged()</signal>
   <receiver>ComboDialog</receiver>
   <slot>updateGui()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboGroup</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ComboDialog</receiver>
   <slot>updateGui()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNewGroup</sender>
   <signal>clicked()</signal>
   <receiver>ComboDialog</receiver>
   <slot>onActionNewGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>ComboDialog</receiver>
   <slot>onActionOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onActionOk()</slot>
  <slot>updateGui()</slot>
  <slot>onActionNewGroup()</slot>
 </slots>
</ui>
